draft-irtf-cfrg-vdaf.md

```diff
@@ -4634,11 +4642,32 @@ We also stress that even if the Idpf is not extractable, Poplar1 guarantees
 that every client can contribute to at most one prefix among the ones being
 evaluated by the helpers.
 
-## Choosing the Number of Proofs to Use for Prio3 {#security-multiproof}
-
-> TODO Add guidance for choosing `PROOFS` ({{multiproofs}}) for Prio3.
- In particular when we go for a smaller field for a given circuit. See
-[this](https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/177) for details.
+## Choosing the Field Size {#security-multiproof}
+
+Prio3 and other systems built from FLPs ({{flp-generic}} in particular) may
+benefit from choosing a field size that is as small as possible. Generally
+speaking, a smaller field results in lower communication and storage costs.
+Care must be taken, however, since a smaller field also results in degraded (or
+even vacuous) robustness.
+
+Different variants of Prio3 ({{prio3}}) use different field sizes: Prio3Count
+uses Field64; but Prio3Sum, Prio3SumVec, and Prio3Histogram use Field128, a
+field that is twice as large as Field64. This is due to the use of joint
+randomness ({{flp}}) in the latter variants. Joint randomness allows for more
+flexible circuit design (see {{flp-generic-overview-extensions}}), but opens up
+Prio3 to precomputation attacks, which the larger field mitigates. (See
+{{DPRS23}}, Theorem 1.) Note that privacy is not susceptible to such attacks.
+
+Another way to mitigate this issue (or improve robustness in general) is to
+generate and verify multiple, independent proofs. (See {{multiproofs}}.) For
+Prio3, the `PROOFS` parameter controls the number of proofs (at least one) that
+are generated and verified.
+
+In general, Field128 is RECOMMENDED for use in Prio3 when the circuit uses
```

Comment:
should we add a section to quantify the risk of using 2 PROOFS vs 3?

Reply:
here soundness is not mentioned, while it was in #multiproofs. Should we mention the relationship between soundness and robustness (which is the overall goal) here?

Reply:
Can you clarify: 2 proofs for Field128 or Field64?

Connecting soundness and robustness is a good idea, will do.

Reply:
2 proofs for Field64.
There were many discussions in issues for robustness analysis, especially 2 proofs vs 3. Would be good to tell reader what are the risks involved using PROOFS == 2. So we can justify why "`PROOFS` MUST be set to at least `3`", instead of "SHOULD be"

Reply:
Below we say "breaking robustness for `PROOFS == 2` is feasible, if impractical".

I mean to use the terms "feasible" and "impractical" the way they are generally meant in the crypto literature:  An attack is "feasilbe' if it can be pulled off, given enough resources. However the amount of resources it takes to pull off means we may not see the attack happen in practice (costs too much money, too much memory, or too much time).

Reply:
Added note about robustness versus soundness.